tests/test_performance.py

```diff
@@ -0,0 +1,16 @@
+from lona.html import Tr, Td
+
+
+class TestPerformance:
+
+    def test_number_of_serialize_calls(self, mocker):
+
+        from lona.html import Node
+
+        spy = mocker.spy(Node, "_serialize")
```

Comment:
Because of the double-quotes, the linter is unhappy here

Reply:
Fixed in ec0315a43357b753d9ac2a57b0200bba0d6aee68